config/initializers/global_functions.rb

```diff
@@ -10,10 +10,18 @@ def production?
   Rails.env.production?
 end
 
+def staging?
+  Rails.env.production?
+end
+
 def development_or_test?
   development? || test?
 end
 
 def development_or_production?
   development? || production?
+end
+
+def staging_or_production?
```

Comment:
call it `production_or_staging? - you should care/think about the most important thing first.